openshift-install-powervs

```diff
@@ -601,13 +601,18 @@ function precheck_input {
     [[ -z "$SERVICE_INSTANCE_ID" ]] && error "Required input variable 'service_instance_id' not found"
     debug_switch
     VAR_IBMCLOUD_API_KEY=$(read_varfile "ibmcloud_api_key")
+    VAR_RHEL_SUBS_PASS=$(read_varfile "rhel_subscription_password")
     debug_switch
     VAR_PULL_SECRET_FILE=$(read_varfile "pull_secret_file")
     VAR_PRIVATE_KEY_FILE=$(read_varfile "private_key_file")
     VAR_PUBLIC_KEY_FILE=$(read_varfile "public_key_file")
+    VAR_RHEL_SUBS_USER=$(read_varfile "rhel_subscription_username")
   fi
 
   debug_switch
+  if [ -n "$VAR_RHEL_SUBS_USER" ] && [ -z "$RHEL_SUBS_PASSWORD" ] && [ -z "$VAR_RHEL_SUBS_PASS" ]; then
```

Comment:
The above case will mean all the three variables will be needed. Is it intended ?

Reply:
Yes, we intend to error out in case `rhel_subscription_password` is not specified in the file and neither exported when the `rhel_username` is already set.

Reply:
It is saying if user is set but password is not set in var or env.